pkg/scheduler/scheduler.go

```diff
@@ -585,7 +585,7 @@ func (sched *Scheduler) scheduleOne(ctx context.Context) {
 			if err := sched.SchedulerCache.ForgetPod(assumedPod); err != nil {
 				klog.Errorf("scheduler cache ForgetPod failed: %v", err)
 			}
-			sched.recordSchedulingFailure(fwk, assumedPodInfo, fmt.Errorf("Binding rejected: %v", err), SchedulerError, "")
+			sched.recordSchedulingFailure(fwk, assumedPodInfo, fmt.Errorf("binding rejected: %v", err), SchedulerError, "")
```

Comment:
Would you mind also replacing `%v` with `%w` for errors? Ref #94695

Reply:
The `%v` has been replaced with `%w`, and its corresponding test is fixed also. But the CI result shows "not something we can merge".